app/controllers/ListEventsController.js

```diff
@@ -9,10 +10,21 @@ GameUpApp.controller('ListEventsController', ['$scope', '$location', function($s
     return query.find();
   };
 
+  $scope.retrievePromotedEventList = function(){
```

Comment:
What happens when there is 20 promoted events lets say, and only 15 spots for promoted events? Would this be a job for the controller to handle this logic?